src/Models/Behaviors/HasPosition.php

```diff
@@ -19,7 +19,7 @@ protected function setToLastPosition()
 
     protected function getCurrentLastPosition()
     {
-        return ((int) static::max('position'));
+        return ((int) static::max("{$this->getTable()}.position"));
```

Comment:
@antonioribeiro I was about to approve this but I realized this isn't a scope and shouldn't be necessary here, correct?

Reply:
Hey @ifox , this isn't a scope, yes, but it's a query, and if we have a global scope on this model joining a bunch of other tables, when we try to get the last position, the `position` column will have no alias and may clash wish some other table. I added it later because it happened to me during a save event. I also prevented by allowing the scope only if the application is running on frontend, but other people might not be able to do this.